README.md

```diff
@@ -51,6 +51,79 @@ Be patient when OpenOCD is flashing. Typically, it takes about 55 seconds.
 Use `make help` to get detailed build targets.
 
 
+# Bug Fixes
```

Comment:
I am not sure if it makes sense to call it "Bug fixes" since we shall improve the build scripts and the prerequisites first.